src/test/java/com/radixdlt/serialization/mapper/RadixObjectMapperConfiguratorTest.java

```diff
@@ -0,0 +1,88 @@
+package com.radixdlt.serialization.mapper;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import com.radixdlt.serialization.ClassScanningSerializationPolicy;
+import com.radixdlt.serialization.DsonOutput;
+import com.radixdlt.serialization.DsonOutput.Output;
+import com.radixdlt.serialization.Serialization;
+import com.radixdlt.serialization.SerializerConstants;
+import com.radixdlt.serialization.SerializerDummy;
+import com.radixdlt.serialization.SerializerId2;
+import com.radixdlt.serialization.core.ClasspathScanningSerializerIds;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class RadixObjectMapperConfiguratorTest {
+	private final Serialization serialization = Serialization.create(
+			ClasspathScanningSerializerIds.create(),
+			new ClassScanningSerializationPolicy(ImmutableList.of(OrderedPropertyBean.class)) {
+			}
+	);
+
+	@Test
+	public void propertyOrderingIsCorrect() {
+		var inputBean =
+				new OrderedPropertyBean("b", "z", "a", "x").propertyC("c");

```

Comment:
I think this would be easier to comprehend if we didn't use the same string values as the field names, eg used "1" for field "a".

Also some comments explaining the output comparison would be useful for folks to determine correctness.

Reply:
Changed values and added comments to make it more clear.